src/gui/widgets/window.hpp

```diff
@@ -221,8 +221,8 @@ class window : public panel, public top_level_drawable
 	/**
 	 * Requests to close the window.
 	 *
-	 * At the moment the request is always honored but that might change in the
-	 * future.
+	 * This request is not always honored, and so callers must account for the window remaining open.
```

Comment:
Is it "not always honored" or "not always honored immediately"? As in, is one call always enough to close the window, although it might take time for it to be processed?

Reply:
Well, the close the campaign credits window does close on its own with this PR applied, so I suppose it's "not always honored immediately".

Reply:
Updated.